integration/kubernetes/runtimeclass_workloads/pod-liveness.yaml

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    test: liveness
+  name: liveness-exec
+spec:
+  runtimeClassName: kata
+  containers:
+  - name: liveness
+    image: busybox
+    args:
+    - /bin/sh
+    - -c
+    - touch /tmp/healthy; echo "Check status"; sleep 6; rm -rf /tmp/healthy; echo "Check dead"; sleep 12
```

Comment:
These `sleep` values seem pretty specific - is there any way to avoid sleeping to make this more robust?